src/fsharp/pars.fsy

```diff
@@ -195,10 +195,10 @@ let rangeOfLongIdent(lid:LongIdent) =
 %} 
 
 // Producing these changes the lex state, e.g. string --> token, or nesting level of braces in interpolated strings
-%token <byte[] * ParseHelpers.LexerContinuation> BYTEARRAY
-%token <string * ParseHelpers.LexerContinuation> STRING 
-%token <string * ParseHelpers.LexerContinuation> INTERP_STRING_BEGIN_END
-%token <string * ParseHelpers.LexerContinuation> INTERP_STRING_BEGIN_PART 
+%token <byte[] * bool * ParseHelpers.LexerContinuation> BYTEARRAY
+%token <string * bool * bool * ParseHelpers.LexerContinuation> STRING 
```

Comment:
Could we use the syntax string/byte kind types instead of bool?

Reply:
Done!